File: pinyougou_parent/pinyougou_seckill_interface/src/main/java/com/pinyougou/seckill/service/SeckillService.java
package com.pinyougou.seckill.service;

import com.pinyougou.pojo.TbSeckillGoods;

import java.util.List;

public interface SeckillService {
    //从缓存中拿到所有秒杀商品
    List<TbSeckillGoods> findAllSeckillGoodsFromRedis();
    //通过秒杀商品ID从缓存中拿到该秒杀商品
    TbSeckillGoods findOneSeckillGoodsFromRedis(Long seckillGoodsId);
    //通过秒杀商品ID,用户ID 保存秒杀订单
    void saveSeckillOrder(Long seckillGoodsId, String userId);
}

File: pinyougou_parent/pinyougou_seckill_web/src/main/webapp/js/service/seckillService.js
app.service("seckillService",function ($http) {

    this.findSeckillList=function () {
      return  $http.get("./seckill/findSeckillList");
    }

    this.findOne=function (seckillGoodsId) {
      return  $http.get("./seckill/findOne/"+seckillGoodsId);
    }

    this.saveSeckillOrder=function (seckillGoodsId) {
      return  $http.get("./seckill/saveSeckillOrder/"+seckillGoodsId);
    }


})